app/views/home/blog.html.erb

```diff
@@ -0,0 +1 @@
+this is a blog

```

Comment:
Can we actually remove the blog stuff?  Even their old website had nothing under the blog section.